uitools/toolkitwidgets/toolkitwidgets.pri

```diff
@@ -13,22 +13,24 @@
 # limitations under the License.
 include($$PWD/../common/common.pri)
 
-QT += widgets webenginewidgets svg
+unix:win32:macx {
```

Comment:
will android trigger the `unix` section? I recall we've had to make exceptions for that in the past.

Also, presumably we should throw an error or something if someone tries to use this on Android.

I don't think we should allow this to be included by platforms that should not use it, and just have it silently do nothing.